src/applications/ivc-champva/10-10D/containers/IntroductionPage.jsx

```diff
@@ -62,12 +62,12 @@ const IntroductionPage = props => {
           <br />
           You may also need to submit supporting documents, like copies of your
           other health insurance cards or proof of school enrollment.
-          {/* Link coming soon: */}
-          {/* 
-          <br />
-          <br />
-          <VaLink text="Find out which documents you’ll need to apply for CHAMPVA" />
-          */}
+          <p>
+            <va-link
+              href="https://www.va.gov/family-and-caregiver-benefits/health-and-disability/champva/#supporting-documents-for-your-"
+              text="Find out which documents you’ll need to apply for CHAMPVA"
```

Comment:
Despite looking like the href has been cut off, this is the actual link to a static Drupal page. Apparently page anchors generated off of headers get capped at a certain length.